bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RetryingReadRowsOperation.java

```diff
@@ -169,8 +171,11 @@ public void onMessage(ReadRowsResponse message) {
 
     ByteString previouslyProcessedKey = rowMerger.getLastCompletedRowKey();
 
+    operationSpan.addAnnotation("Got a response");
     // This may take some time. This must not block so that gRPC worker threads don't leak.
     rowMerger.onNext(message);
+    operationSpan.addAnnotation("Processed Response", ImmutableMap.of("rowCount",
+      AttributeValue.longAttributeValue(rowMerger.getRowCountInLastMessage())));
```

Comment:
How does this work with multiple responses? Do they accumulate or overwrite each other?

Reply:
I'll add docs.  This is for a single response.  We probably should also have a different accumulated value.